etc/proto/run

```diff
@@ -1,7 +1,5 @@
 #!/bin/sh
 
-set -x
```

Comment:
This was a small unrelated change. Basically this omits the large output from `make proto` so that you can see errors. I've been bitten by this a few times, so it seemed handy to include. If we want to omit it, that's ok.